app/components/Channels/ChannelCloseDialog.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Flex } from 'rebass'
+import { FormattedMessage, injectIntl, intlShape } from 'react-intl'
+import { withFieldState } from 'informed'
+import Delete from 'components/Icon/Delete'
+import { Dialog, Text, Heading, Button, Checkbox, Form, DialogOverlay } from 'components/UI'
+import { useCloseOnUnmount } from 'components/Util/hooks'
+import messages from './messages'
+
+const DialogWrapper = ({ intl, isForceClose, isOpen, onClose, onCancel, csvDelay }) => {
+  useCloseOnUnmount(isOpen, onCancel)
+
+  if (!isOpen) {
+    return null
+  }
+  const checkboxFieldName = 'actionACK'
+  // bind button disabled state to a form field
+  const CloseChannelButton = withFieldState(checkboxFieldName)(({ fieldState, ...rest }) => (
+    <Button disabled={isForceClose && !fieldState.value} {...rest} />
+  ))
+
+  const buttons = (
+    <>
+      <CloseChannelButton variant="danger" type="submit">
+        <FormattedMessage
+          {...(isForceClose
+            ? messages.close_channel_dialog_force_close_text
+            : messages.close_channel_dialog_close_text)}
+        />
+      </CloseChannelButton>
+      <Button variant="secondary" type="button" onClick={onCancel}>
+        <FormattedMessage {...messages.close_channel_dialog_cancel_text} />
+      </Button>
+    </>
+  )
+
+  const header = (
+    <Flex flexDirection="column" alignItems="center" alignSelf="center">
+      <Delete color="#e8383a" width={72} height={72} />
```

Comment:
What is this hardcoded colour?

Reply:
Strange stuff. Probably inherited from delete dialog. So has to be updated there too

Reply:
updated